lib/diagnostic.js

```diff
@@ -6,26 +6,40 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizedWords = function getNormalizedWords(stringInput) {
+  let words = stringInput.split(/\s+/);
+  for(let i=0; i<words.length; i++){
+    words[i] = words[i].replace(/\W+/, '').toUpperCase();
+  }
+  return words;
 };
 
```

Comment:
On line 9 you call your function normalizedWords and then you write function **get**NormalizedWords. This is OK, but you need to be careful when you do this. _Inside_ the function, it is referred to as getNormalizedWords, outside it is referred to as normalizedWords. Just a sticky thing, I recommend getting in the habit of naming them the same.